Addition of statechart and signals2 to boost_library

sanjeevcvh opened the pull request:
Addition of statechart and signals2 to boost_library

sanjeevcvh committed:
Addition of statechart and signals2 to boost_library

Addition of statechart and signals2 to boost_library

iceboy233 commented on boost_libraries.bzl:
```diff
@@ -752,6 +752,59 @@ BOOST_LIBRARIES = [
     ),
+    
+    boost_library(
+        name = "conversion",
```
Could you please sort the `boost_library` entries in alphabet order?

sanjeevcvh replied:
Agreed and will do the changes

sanjeevcvh committed:
Updated the inclusion of libraries statechart and signals2in the alphabetical order 

Updated the inclusion of libraries statechart and signals2in the alphabetical order

iceboy233 commented on boost_libraries.bzl:
```diff
@@ -625,6 +661,24 @@ BOOST_LIBRARIES = [
     ),
 
+    boost_library(
+        name = "statechart",
```
nit: statechart < static_assert

sanjeevcvh replied:
done the changes

iceboy233 reviewed (approved):
Note: I was thinking to add build test, to ensure build success, as new boost versions can change dependencies, which was seen in the past. Previously this was not a problem as all libraries are used by my repositories and I can use them to test boost upgrades. Let me know if you have any idea on how to implement such test. Thanks!

sanjeevcvh committed:
statechart before the static_Assert

statechart before the static_Assert

iceboy233 commented:
Thanks!
